### ISSUE TITLE
error

### ISSUE THREAD

**Issue by mandar2129** [dilip29/Summer-Training-Python-development#3]
whenever i create a new team and try to calculate its score..it gives typeError nontype object is not subsciptable line no 202,in dlgscore.py file...and it popups python has stopped working please address this issue.

**Comment by Franchis94** [comment]
Did you checked the fantasy.db file....is the new team added there..?

**Comment by mandar2129** [comment]
yes i checked,new team is added there in the database file.

**Comment by ghost** [comment]
whenever i create a new team and try to calculate its score..it gives typeError nontype object is not subsciptable line no 202,in dlgscore.py file...and it popups python has stopped working please address this issue.

**Comment by rithickb068** [comment]
Yeah even I have the same problem please resolve that, non type object is not subsciptable
